app/docs/md/learn/practices/testing.md

```diff
@@ -85,3 +85,40 @@ test('Shut down local server', async t => {
 })
 ```
 
+## Component testing
+
+Running component tests allow you to mount, render and interact with a single Enhance component in isolation. To ensure that test conditions are as close as possible to the real world, we recommend to run them in actual browser rather than using e.g. [JSDOM](https://github.com/jsdom/jsdom) which is just a JavaScript implementation of web standards browser offer. To run Enhance component tests you can use [WebdriverIO](https://webdriver.io/) as a browser automation framework. It is based on the [WebDriver](https://www.w3.org/TR/webdriver/) protocol which is a browser automation web standard which guarantees that your tests interact with your components as closely as possible to how a user would.
```

Comment:
Just a couple tweaks for readability:

```suggestion
Running component tests allow you to mount, render and interact with a single Enhance component in isolation. To ensure that test conditions are as close as possible to the real world, we recommend to run them in actual browser rather than using e.g. [JSDOM](https://github.com/jsdom/jsdom) which is just a JavaScript implementation of web standards provided by browsers. To run Enhance component tests you can use [WebdriverIO](https://webdriver.io/) as a browser automation framework. It is based on the [WebDriver](https://www.w3.org/TR/webdriver/) protocol, a browser automation web standard which guarantees that your tests interact with your components as closely as possible to how a user would.
```

Also, I'd suggest placing each sentence in its own line (which doesn't affect wrapping in markdown, i.e. the final rendered document is identical), so that any future changes can be highlighted more conveniently by diff rendering engines.

Reply:
Nice tip, will do that!